README_zh-CN.md

```diff
@@ -70,42 +70,22 @@ MMClassification 是一款基于 PyTorch 的开源图像分类工具箱，是 [O
 ### 主要特性
 
 - 支持多样的主干网络与预训练模型
-- 支持配置多种训练技巧
+- 支持配置多种训练策略和技巧
 - 大量的训练配置文件
 - 高效率和高可扩展性
 - 功能强大的工具箱
 
 ## 更新日志
 
-2022/12/30 发布了 v1.0.0rc5 版本
+2022/12/30 发布了 v1.0.0rc0 版本
 
-- 支持了**EVA**, **RevViT**, **EfficientnetV2**, **CLIP**, **TinyViT** 和 **MixMIM** 等骨干网络结构
-- 复现了 ConvNeXt 和 RepVGG 的训练精度。
-- 支持了 **多任务** 训练和测试，详见 [#1229](https://github.com/open-mmlab/mmclassification/pull/1229)
-- 支持了测试时增强（TTA），详见 [#1161](https://github.com/open-mmlab/mmclassification/pull/1161)
+- 整和来自 MMSelfSup 的自监督学习算法
 
-2022/12/06 发布了 v1.0.0rc4 版本
+这个版本引入一个全新的，可扩展性强的训练和测试引擎，但目前仍在开发中。欢迎根据 [文档](https://mmclassification.readthedocs.io/zh_CN/1.x/) 进行试用。
 
-- 更新了主要 API 接口，用以方便地获取 MMClassification 中预定义的模型。详见 [#1236](https://github.com/open-mmlab/mmclassification/pull/1236)。
-- 重构 BEiT 主干网络结构，并支持 v1 和 v2 模型的推理。
-
-2022/11/21 发布了 v1.0.0rc3 版本
-
-- 添加了 **Switch Recipe Hook**，现在我们可以在训练过程中修改数据增强、Mixup设置、loss设置等
-- 添加了 **TIMM 和 HuggingFace** 包装器，现在我们可以直接训练、使用 TIMM 和 HuggingFace 中的模型
-- 支持了检索任务
-- 复现了 **MobileOne** 训练精度
-
-2022/8/31 发布了 v1.0.0rc0 版本
-
-这个版本引入一个全新的，可扩展性强的训练和测试引擎，但目前仍在开发中。欢迎根据[文档](https://mmclassification.readthedocs.io/zh_CN/1.x/)进行试用。
-
-同时，新版本中存在一些与旧版本不兼容的修改。请查看[迁移文档](https://mmclassification.readthedocs.io/zh_CN/1.x/migration.html)来详细了解这些变动。
-
-新版本的公测将持续到 2022 年末，在此期间，我们将基于 `1.x` 分支进行更新，不会合入到 `master` 分支。另外，至少
-到 2023 年末，我们会保持对 0.x 版本的维护。
-
-发布历史和更新细节请参考 [更新日志](https://mmclassification.readthedocs.io/zh_CN/1.x/notes/changelog.html)
+同时，新版本中存在一些与旧版本不兼容的修改。请查看 [迁移文档](https://mmclassification.readthedocs.io/zh_CN/1.x/migration.html) 来详细了解这些变动。
+ 
+发布历史和更新细节请参考 [更新日志](https://mmclassification.readthedocs.io/zh_CN/1.x/notes/changelog.html)。
```

Comment:
1.x  ->  pretrain ?

Reply:
will be updated after readthedocs being ready